test/functional/terminal/channel_spec.lua

```diff
@@ -198,4 +198,64 @@ describe('no crash when TermOpen autocommand', function()
     ]]}
     assert_alive()
   end)
+
+  it('nvim_open_term({force_crlf=true}) converts newlines', function()
+    local buf = meths.create_buf(false, true)
+    local win = meths.get_current_win()
+    local term = meths.open_term(buf, {force_crlf = true})
+    screen:try_resize(8, 10)
+    meths.win_set_buf(win, buf)
+    meths.chan_send(term, 'here\nthere\n')
+    screen:expect{grid=[[
+      ^here        |
+      there       |
+                  |
+      {0:~           }|
+      {0:~           }|
+      {0:~           }|
+      {0:~           }|
+      {0:~           }|
+      {0:~           }|
+                  |
+    ]]}
+    meths.chan_send(term, 'foo\r\nbar\n')
```

Comment:
can't all (or most) of these cases be sent in one chan_send, then tested with a single screen:expect?

also need a test case for `force_crlf=false`.

Reply:
can be in one channel